shelley/chain-and-ledger/formal-spec/update.tex

```diff
@@ -94,6 +96,11 @@ \subsection{Protocol Parameter Update Proposals}
 \item In the case of \var{slot} being greater than or equal to
   $\fun{firstSlot}~((\epoch{slot}) + 1) - \fun{SlotsPrior}$, there is
   a \emph{PPUpdateTooLate} failure.
+\item In the case that the epoch number in the signal is not the current epoch,
+  there is a \emph{PPUpdateNoEpoch} failure.
+  The reason for requiring protocol parameter updates to specify the target
+  epoch is that otherwise there are transactions which change from being invalid
+  to being valid when the epoch number increases, which is better to avoid.
```

Comment:
is the reason for this documented somewhere ?

Reply:
you mean besides here? or that I should explain more?